### ISSUE TITLE
localhost/pb does not work

### ISSUE THREAD

**Issue by raybeatsofficial** [redteamsecurity/PlugBot-Plug#10]
Hi,

my problem is that i can't open localhost/pb and a tried it with using my ip address, same problem:

Not Found

The requested URL /pb was not found on this server.
Apache/2.4.10 (Raspbian) Server at myipaddress Port 80

Please help me as soon as possible :(

**Comment by raybeatsofficial** [comment]
i also installed C2C, still not working..

**Comment by Void156** [comment]
Did you install the PlugBot files in a folder named pb?

**Comment by raybeatsofficial** [comment]
noo i installed every file that is neede for the plugbot in /var/www/    the PlugBot folder is in there and the PlugBotC2C folder

**Comment by raybeatsofficial** [comment]
damn i didn't wanted to close

**Comment by Void156** [comment]
Copy the contents of The Plugbot folder in /var/www/

**Comment by raybeatsofficial** [comment]
yeah i did already done, still not working i can connect to index.php/home/setup but not so /pb

**Comment by Void156** [comment]
Well you do not need to connect to /pb/ . index.php/home/setup is all you need. /pb/ is not even a folder

**Comment by raybeatsofficial** [comment]
but how can i install some scripts then

**Comment by raybeatsofficial** [comment]
in the video Getting started with PlugBot #2/3 he connectin to /pb right

**Comment by raybeatsofficial** [comment]
i can't open the dropzone url, my dropzone url is /pb nothing more at the end

**Comment by MikeBrawley** [comment]
@raybeatsofficial  I think I figured this out and will test it tonight.  Plugbot-plug is only one part of a plugbot deployment. You also need to setup https://github.com/redteamsecurity/PlugBot-C2C offsite in a seperate location.  The plugbot-plug connects back to the C2C and C2C is the website you connect to do all of your activities.  Plugbot-Plug is simply a connection outbound that stays open to the C2C for you to get the backdoor into the location of the Plugbot-plug.  

Also I found this article useful to get plugbot to work with PHP  https://www.helloitsliam.com/2017/09/14/note-to-self-plugbot-on-raspberry-pi-raspbian-php7-fixes/
